website/docs/guides/orchestration/webhooks/serverless-datadog.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Create Datadog events from dbt Cloud results"
+id: webhooks-guide-serverless-datadog
+slug: serverless-datadog
+description: Configure a serverless app to add Datadog logs
+---
+
+This guide will teach you how to build and host a basic Python app which will add dbt Cloud job events to Datadog. To do this, when a dbt Cloud job completes it will create a log entry for each node that was run, containing all information about the node provided by the [Metadata API](docs/dbt-cloud-apis/metadata-schema-models).
```

Comment:
the metadata api is showing a bad link
<img width="818" alt="image" src="https://user-images.githubusercontent.com/18452766/221981475-88f4c708-a1a6-451b-81e9-09fbf79add7f.png">